packages/common/sagas/workflow.js

```diff
@@ -295,6 +296,9 @@ function* initTeams() {
     const teamById = convertTeamsToEntity(teams);
 
     yield put(addTeamsBatch(teamById));
+    yield call(openCurrentVaultSaga);
+
+    yield put(finishIsLoading());
```

Comment:
Do not use the finishIsLoading and the openCurrentVaultSaga cause the openTeamVaultSaga should have a call of the finishIsLoading.

Reply:
1. Returned back finishIsLoading  into openTeamVaultSaga.
2. Removed finishIsLoading and openCurrentVaultSaga from initTeams.